Library/Homebrew/cmd/list.rb

```diff
@@ -20,13 +20,28 @@
 
 require "metafiles"
 require "formula"
+require "cli_parser"
 
 module Homebrew
   module_function
 
   def list
+    @args = CLI::Parser.parse do
+      switch "--unbrewed"
+      switch "--pinned"
+      switch "--versions"
+      switch "--full-name"
+      switch "--multiple"
```

Comment:
`switch "--multiple", required_for: "--versions"`

Reply:
Don't you mean `depends_on`? `--multiple` is optional.

Reply:
Ops.. I got that wrong, you're right.